doc/modules/model_evaluation.rst

```diff
@@ -215,6 +215,23 @@ the following two rules:
   Again, by convention higher numbers are better, so if your scorer
   returns loss, that value should be negated.
 
+While defining the custom scoring function alongside the calling function 
```

Comment:
`the cross validation function` would be more precise than `the calling function`

Reply:
> Also as rmse is not defined anywhere in the example and as the example is not run, 
why do you think of using custom_scoring_function or custom_scorer instead of rmse?

I don't think I understood your comment clearly. 

re: "as the example is not run": The example is being skipped from 
executing because we felt adding a module just for one line in the documentation 
would be an overkill. 

Regarding the rest: `rmse` is only a placeholder-name for a custom function which is 
not available as a custom scorer via sklearn, directly. The line above the example
and the context of this documentation section makes it clear, don't you think ?
> the cross validation function would be more precise than the calling function

The custom scorer can be used with any model which provides the `n_jobs` parameter so I think the calling function need not always be the `cross validation function` only. I feel "Calling function" would meet that requirement better.